ElasticaBundle/Command/OrchestraCreateSchemaCommand.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace OpenOrchestra\ElasticaBundle\Command;
+
+use LogicException;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class OrchestraCreateSchemaCommand
+ */
+class OrchestraCreateSchemaCommand extends ContainerAwareCommand
```

Comment:
of course, but I think that this will come with the usage.

This pr is a first functionning indexation for the content, when we add the nodes, we would modify add some parameters